src/gui/theme.js

```diff
@@ -103,3 +105,7 @@ export function getNavigationMenuBg(): string {
 export function getNavigationMenuIcon(): string {
 	return theme.navigation_menu_icon || theme.navigation_button_icon
 }
+
+export function getColouredTutanotaLogo(): string {
+	return getLogoSvg(theme.content_accent, isColorLight(theme.content_bg) ? '#4a4a4a' : '#c5c7c7')
```

Comment:
Don't hardcode colors, you can get the theme you need and get the color from there

Reply:
these values don't exist on the theme but we can just grab the constants

Reply:
yeah that would be okay